src/Owin.Security.Providers.WSO2/WSO2AuthenticationExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.Owin;
+using Owin;
+using System;
+
+namespace Owin.Security.Providers.WSO2
+{
+    public static class WSO2AuthenticationExtensions
+	{
+		public static IAppBuilder UseWSO2Authentication(this IAppBuilder app, WSO2AuthenticationOptions options)
+		{
+			if (app == null)
+				throw new ArgumentNullException(nameof(app));
+			if (options == null)
+				throw new ArgumentNullException(nameof(options));
+
+			app.Use(typeof(WSO2AuthenticationMiddleware), app, options);
+
+			return app;
+		}
+
+		public static IAppBuilder UseWSO2Authentication(this IAppBuilder app, string baseUrl, string clientId, string clientSecret)
```

Comment:
So, as I know it wso2 is an identity server? 

I understand the user must pass a base url to their identity server, but could we add a comment for the parameter giving the user guidance as to what that should look like?

I assume its just https://mywso2server ?? but I have no idea, do they need to include the protocol? what about a trailing slash?